packages/palette/src/elements/Clickable/Clickable.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react"
```

Comment:
Will want to create an iOS stub `Clickable.ios.tsx` so that Eigen doesn't automatically import .js, encounter possible web content, and break. Eventually, if this is shared / extended, we can collapse these two files into one.

Reply:
Yeah there's an empty one in here

Reply:
cool, just wanted to be sure 👍